interface/forms/newpatient/C_EncounterVisitForm.class.php

```diff
@@ -0,0 +1,723 @@
+<?php
+
+/**
+ * Controller class for the encounter form (new and view) scripts.  It is responsible for rendering the encounter form.
+ * I keep similar class names as to other forms.  This is to keep consistency in the codebase.  At some point
+ * we may want to move all of these into the src folders but as these form folders can be moved around and are
+ * supposed to be self-contained, I'm leaving this here.
+ * TODO: investigate moving this into the src/ folder.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Ranganath Pathak <pathak@scrs1.org>
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2019 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2019 Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2025 Mountain Valley Health <mvhinspire@mountainvalleyhealthinc.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @license   There are segments of code in this file that have been generated via Claude.ai and are licensed as Public Domain.  They have been marked with a header and footer.
+ */
+
+namespace OpenEMR\Forms\NewPatient;
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Acl\AclExtended;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Core\Kernel;
+use OpenEMR\Events\Core\TemplatePageEvent;
+use OpenEMR\Services\FacilityService;
+use OpenEMR\Services\UserService;
+use OpenEMR\Services\ListService;
+use OpenEMR\OeUI\RenderFormFieldHelper;
+use OpenEMR\Common\Database\QueryUtils;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Twig\TwigFunction;
+use OpenEMR\Billing\MiscBillingOptions;
+use Twig\Environment;
+use sqlStatement;
+use sqlFetchArray;
```

Comment:
not fully alphabetized. Also haven't seen the use command for functions before; just making sure meant to be there.